src/main/scala/com/adobe/api/platform/runtime/metrics/OpenWhiskEvents.scala

```diff
@@ -30,15 +32,22 @@ object OpenWhiskEvents extends SLF4JLogging {
 
   def start(config: Config)(implicit system: ActorSystem,
                             materializer: ActorMaterializer): Future[Http.ServerBinding] = {
+    Kamon.reconfigure(config)
+    val prometheusReporter = new PrometheusReporter()
+    Kamon.addReporter(prometheusReporter)
+
     val metricConfig = loadConfigOrThrow[MetricConfig](config, "user-events")
-    val port = metricConfig.port
-    val recorders = if (metricConfig.enableKamon) Seq(PrometheusRecorder, KamonRecorder) else Seq(KamonRecorder)
-    //Make KamonConsumer configurable
-    val kamonConsumer = EventConsumer(eventConsumerSettings(defaultConsumerConfig(config)), recorders)
-    val api = new PrometheusEventsApi(kamonConsumer, PrometheusRecorder)
+
+    val prometheusRecorder = PrometheusRecorder(prometheusReporter)
+    val recorders = if (metricConfig.enableKamon) Seq(prometheusRecorder, KamonRecorder) else Seq(prometheusRecorder)
```

Comment:
There was a bug in master where by default the java client based Prometheus reporter was not getting used and instead Kamon one was getting used.. So changed the else to default to java prometheus client